### ISSUE TITLE
Non-Fatpack sd card build failed

### ISSUE THREAD

**Issue by rootzoll** [raspiblitz/raspiblitz#3249]
When trying to build a sd card with no fatpack `-f 0` (without API/WebUI) using the following command on a fresh RaspiOS 64-bit:

`wget --no-cache https://raw.githubusercontent.com/rootzoll/raspiblitz/dev/build_sdcard.sh && sudo bash build_sdcard.sh -f 0 -b dev`

It failes on this error:

```
*** Adding deb.torproject.org keyring ***
gpg: no valid OpenPGP data found.
! FAIL: Was not able to import deb.torproject.org key
```

@openoms this is not v1.8.0 critical - but should be fixed soon after v1.8.0 release in v1.8 branch.

**Comment by openoms** [comment]
Likely times out as Tor is so bad nowadays. Will add a clearnet fallback to pull the keys.

**Comment by rootzoll** [comment]
@openoms let me know when this is in - its the last issue to close post-v1.8.0 release.

**Comment by openoms** [comment]
fixed in 35607331 both in dev and v1.8
